test/test_nestedtensor.py

```diff
@@ -274,6 +274,13 @@ def test_to_padded_tensor_on_empty_tensor(self):
         nt = torch.nested_tensor([])
         empty = nt.to_padded_tensor(4)
         self.assertEqual(empty, torch.tensor([]))
+
+    def test_nested_namespace(self):
+        nt = torch.nested_tensor([torch.randn(2, 3), torch.randn(4, 5)])
+        result = nt.to_padded_tensor(4)
```

Comment:
yeah I hope it isn't the pattern duplicate torch.* namespace functions into the aliases because having both feels really wrong

Reply:
This one is method-only. So there is nothing in `torch.*`.

Reply:
Ohh see when then added a function that goes in nested namespace? Do we need that? Should only functional variants go in nested namespace?

Reply:
In general yes because you don't want nested-specific methods on plain Tensor.
But I guess it's too late :p